arrayfire.gemspec

```diff
@@ -0,0 +1,20 @@
+Gem::Specification.new do |gem|
+  gem.name        = 'ArrayFire'
+  gem.version     = '0.0.0'
```

Comment:
Let us keep 0.0... for beta version. Later we can match it with the version of ArrayFire.